Fix the size argument to 'memset'.

Patch by Mark H Weaver <mhw@netris.org>.

--- bitlbee/lib/md5.c.orig	2013-11-27 17:54:54.000000000 -0500
+++ bitlbee/lib/md5.c	2014-03-05 21:39:04.739746093 -0500
@@ -159,7 +159,7 @@ void md5_finish(struct MD5Context *ctx,
 	ctx->buf[2] = cvt32(ctx->buf[2]);
 	ctx->buf[3] = cvt32(ctx->buf[3]);
 	memcpy(digest, ctx->buf, 16);
-	memset(ctx, 0, sizeof(ctx));	/* In case it's sensitive */
+	memset(ctx, 0, sizeof(*ctx));	/* In case it's sensitive */
 }
 
 void md5_finish_ascii(struct MD5Context *context, char *ascii)
